lib/db.js

```diff
@@ -57,7 +58,7 @@ db.create = function (options) {
 
 function Db(options) {
   this.options = options;
-  this._mdb = new mongodb.Db(this.options.name, new mongodb.Server(this.options.host, this.options.port));
+  this.connectionString = this.options.connectionString || "mongodb://" + (this.options.credentials ? this.options.credentials.username + ":" + this.options.credentials.password + "@" : "") + this.options.host + ":" + this.options.port + "/" + this.options.name;
```

Comment:
I think it is nicer to have connectionString then a _mdb object.
But this line may be too long.